Client/game_sa/CVehicleSA.h

```diff
@@ -636,6 +636,7 @@ class CVehicleSA : public virtual CVehicle, public virtual CPhysicalSA
     bool           GetTyresDontBurst() { return GetVehicleInterface()->m_nVehicleFlags.bTyresDontBurst; };
     unsigned short GetAdjustablePropertyValue() { return *reinterpret_cast<unsigned short*>(reinterpret_cast<unsigned long>(m_pInterface) + 2156); };
     float          GetHeliRotorSpeed() { return *reinterpret_cast<float*>(reinterpret_cast<unsigned int>(m_pInterface) + 2124); };
+    float          GetPlaneRotorSpeed() { return *reinterpret_cast<float*>(reinterpret_cast<unsigned int>(m_pInterface) + 0x09C4); };
```

Comment:
I personally like using hexadecimal offset literals here, but the surrounding code uses decimal offset literals. I'm not sure whether we should prefer consistency with the existing code over my style preferences. What do others think about this?

Reply:
personally, I l like to go with hex literals too, I think it's kinda a universal standard to represent something involved with low-level things. but seems the code for heli is using decimal, so there are 2 solution comes up from my mind.
1. Should I change the heli's code to hex? this will make it constant, I guess the majority of the memory address related code uses hex representation in MTA, so make it consitant.
2. Should I stay with the heli's rotor format, using decimals?

What would you suggest? @AlexTMjugador

Reply:
I'd suggest 1, but it'd nice to know what other developers think about it.